deploy/crds/compliance.openshift.io_compliancescans_crd.yaml

```diff
@@ -70,6 +70,9 @@ spec:
                 a specific rule. Note that when leaving this empty, the scan will
                 check for all the rules for a specific profile.
               type: string
+            scanType:
```

Comment:
I thought this went in as part of the platform scan patch, from running `make generate` ??

Reply:
that's just how it turned up. Did a make generate and it changed the place of this :/